src/gurt/telemetry.c

```diff
@@ -2556,10 +2572,13 @@ d_tm_add_ephemeral_dir(struct d_tm_node_t **node, size_t size_bytes,
 	if (rc != 0)
 		D_GOTO(fail, rc);
 
+	if (tm_shmem.ephemeral_dir_lock)
```

Comment:
Are we only concerned with the local setting, and not the flag in the shmem header? If we don't want each shmem region to potentially have its own settings, we should only have the flag in one place.